engine/flink/management/periodic/src/main/scala/pl/touk/nussknacker/engine/management/periodic/PeriodicProcessManager.scala

```diff
@@ -81,10 +81,10 @@ class PeriodicProcessManager(val delegate: ProcessManager,
     }
   }
 
-  private def cancelIfAlreadyDeployed(processVersion: ProcessVersion, user: User): Future[Unit] = {
+  private def cancelIfAlreadyDeployedOrFailed(processVersion: ProcessVersion, user: User): Future[Unit] = {
     findJobStatus(processVersion.processName)
       .map {
-        case Some(s) => s.isDeployed //for periodic status isDeployed=true means deployed or scheduled
+        case Some(s) => s.isDeployed || s.status.isFailed //for periodic status isDeployed=true means deployed or scheduled
```

Comment:
I wonder - do we really have any states where we shouldn't deactivate? I mean, maybe it should be findJobStatus(...).map(_.isDefined)?

Reply:
:+1: